configs/webpack-config-compass/src/externals.ts

```diff
@@ -12,6 +12,8 @@ export const sharedExternals: string[] = [
   'win-export-certificate-and-key',
   'macos-export-certificate-and-key',
   'mongodb-client-encryption',
+  // We externalize react as we want to avoid multiple instances of it.
+  'react',
```

Comment:
Currently `compass-explain-plan` and `compass-aggregations` use `compass-crud`. This works fine in the hot reloading app, however in the testing mocha build we're seeing two react instances which crashes it. We need to externalize react so only the top consumer brings it in. This isn't really the place to do that, we probably want it somewhere in the `index.ts` of this folder. There might be another way to do it too, looking into.